src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java

```diff
@@ -72,6 +78,9 @@ public class SchemaUtils {
             REF_ALLELE_FIELD_NAME, ALT_ALLELE_FIELD_NAME, CALL_GT, CALL_GQ, AS_QUALapprox, QUALapprox, CALL_PL, CALL_AD,
             CALL_PGT, CALL_PID, CALL_PS);
 
+
+    public static final List<String> SAMPLE_PLOIDY_FIELDS = Arrays.asList(CHROMOSOME,SAMPLE_ID,GENOTYPE,PLOIDY);
```

Comment:
when in Rome

```suggestion
    public static final List<String> SAMPLE_PLOIDY_FIELDS = Arrays.asList(CHROMOSOME, SAMPLE_ID, GENOTYPE, PLOIDY);
```

Reply:
I think I had the misfortune of copying from line 72, the first one in the list which happens to also have no spacing between the constants. I was fooled by Rome itself! *shakes fist at sky*